.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,62 @@
+name: Compute Benchmarks
+
+on:
+  workflow_dispatch:
+    inputs:
+      str_name:
+        description: Formatted adapter name
+        type: choice
+        required: true
+        default: 'level_zero'
+        options:
+          - level_zero
+          - level_zero_v2
+      unit:
+        description: Test unit (cpu/gpu)
+        type: choice
+        required: true
+        default: 'gpu'
+        options:
+          - cpu
+          - gpu
+      pr_no:
+        description: PR number (if 0, it'll run on the main)
+        type: number
```

Comment:
shouldn't this be `string` now?

Reply:
This affects the form input. But no, it doesn't have to be a string.